pkg/services/ngalert/state/historian/annotation.go

```diff
@@ -170,15 +170,15 @@ func (h *AnnotationBackend) Query(ctx context.Context, query ngmodels.HistoryQue
 	return frame, nil
 }
 
-func buildAnnotations(rule history_model.RuleMeta, states []state.StateTransition, logger log.Logger) []annotations.Item {
+func BuildAnnotations(rule history_model.RuleMeta, states []state.StateTransition, logger log.Logger) []annotations.Item {
```

Comment:
I don't see any new calls to this outside the historian package, does it need to be made public?